script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -5,13 +5,13 @@ I spent a full two hours in the dining room.
 <Page2>
 "Welcome back, Shiki-sama. Although you did not instruct me to do so, I have prepared your bed."
 <Page3>
-...Or so she says.
+...she says.
 Hisui was quietly making the bed in my room.
 It seems that she had returned a step ahead of me to get things ready.
 <Page4>
 "Ah―――yes, thank you."
 "As for your pajamas, please use the ones over there.
-If you could put your clothes away in the basket and leave them out in the hallway that would be very helpful."
+If you could put your clothes away in the basket and leave them out in the hallway, that would be very helpful."
```

Comment:
drop "away" and "leave it" since she's referring to the basket containing the clothes, not the clothes.